Nos/Views/NoteOptionsButton.swift

```diff
@@ -46,7 +46,7 @@ struct NoteOptionsButton: View {
                         analytics.viewedNoteSource()
                         showingSource = true
                     }
-                    Button(String(localized: .localizable.reportNote), role: .destructive) {
+                    Button(String(localized: .localizable.reportNote)) {
```

Comment:
I miss the reddish-orange since it made it stand out as a serious action but I suppose I missed my chance to bring this up on the design; I see you've followed it.

Reply:
Maybe those are pushed further down to the other dialogs?

Reply:
I still see that effect if I hover though the options. I don't think it is the destructive red, it seems to be the red we usually use as `accent`. I'll check.